### PR TITLE
update readme and move cli related scripts to cli directory

### PR DESCRIPTION
**Pull request by KirillTim**
What do you think? Is it enough description for the current state?
Issue: https://github.com/marco-c/crashsimilarity/issues/2

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,12 +3,22 @@
...
 [![codecov](https://codecov.io/gh/marco-c/crashsimilarity/branch/master/graph/badge.svg)](https://codecov.io/gh/marco-c/crashsimilarity)
 
-## Tests
+### Description
+Crashsimilarity is a tool to cluster slightly different crash reports, reporting the same problem. It is built for mozilla and support crash reports produced by [breakpad](https://chromium.googlesource.com/breakpad/breakpad/).
 
+ :exclamation: The project is in early prototype stage.  :exclamation:
+
+### How does it work
+It use doc2vec for word embedding. Similarity based on [WMD Distance](http://proceedings.mlr.press/v37/kusnerb15.pdf) is calculated.
```
Comment (marco-c) [inline README.md]: `It uses doc2vec for word embedding and [WMD](http://proceedings.mlr.press/v37/kusnerb15.pdf) as a distance metric.`

File: README.md [hunk 2]
```diff
@@ -3,12 +3,22 @@
...
+
+### How to run
+For now only command line interface is supported. Have a look at [cli](https://github.com/marco-c/crashsimilarity/tree/master/crashsimilarity/cli) directory.
```
Comment (marco-c) [inline README.md]: `Have a look at the ...`

File: README.md [hunk 3]
```diff
@@ -3,12 +3,22 @@
 [![Build Status](https://travis-ci.org/marco-c/crashsimilarity.svg?branch=master)](https://travis-ci.org/marco-c/crashsimilarity)
 [![codecov](https://codecov.io/gh/marco-c/crashsimilarity/branch/master/graph/badge.svg)](https://codecov.io/gh/marco-c/crashsimilarity)
 
-## Tests
+### Description
+Crashsimilarity is a tool to cluster slightly different crash reports, reporting the same problem. It is built for mozilla and support crash reports produced by [breakpad](https://chromium.googlesource.com/breakpad/breakpad/).
```
Comment (marco-c) [inline README.md]: `It is built for mozilla and support crash reports produced by [breakpad](https://chromium.googlesource.com/breakpad/breakpad/).` -> `It supports crashes reported to the Mozilla [Socorro crash reporting system](link to socorro home page)`

No need to mention Breakpad, as the project doesn't really depend on the particular software that generates the reports.

Comment (KirillTim) [inline README.md]: Hm, indeed. My point was to tell that for now we are can only work with one particular provider.
I will remove this mention.

Comment (KirillTim) [inline README.md]: > (link to socorro home page)

Should it be link to https://crash-stats.mozilla.com/ or to https://github.com/mozilla/socorro ?

Comment (marco-c) [inline README.md]: Let's use the first one, as it's more user-facing

**Comment by KirillTim** [comment]
Update readme.md and resolve merge confilct.

**Pull request merged**